.buildkite/scripts/run_standard_models.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+## remove failure for testing
+set -euo pipefail
+
+apt-get update
+apt-get install libsasl2-dev
+
+python3 -m venv venv
+. venv/bin/activate
+pip install --upgrade pip setuptools
+pip install -r integration_tests/requirements.txt
+mkdir -p ~/.dbt
+cp integration_tests/ci/sample.profiles.yml ~/.dbt/profiles.yml
+
+db=$1
+echo `pwd`
+cd integration_tests
+dbt deps ## Install all packages needed
```

Comment:
I noticed that you're using the same `run_standard_models.sh` file for all of your pipeline concurrent steps. I think if in the future, we wish to optimize this process -- it may be a good idea to not `dbt deps` all packages and only packages that will be run in that step.

Reply:
I completely agree, I just opted not to do it this time around since that didn't seem to be a blocker and would require some string matching that I was not willing to dive into in this PR haha.

Additionally, the idea behind `run_standard_models.sh` is that all the standard runs (included in this PR) would be held in that file. My idea is to then expand this to other .sh files that test different components. But that is for a different PR 😄

Reply:
Not resolving to keep for later reference.

Reply:
[FR](https://github.com/fivetran/dbt_fivetran_utils/issues/120) opened to address this in the future.